examples/wordpress-mysql/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.7'
 services:
 
   db:
-    image: mysql:5.7
+    image: mysql:8.0
```

Comment:
tiny one, would we good to check whether it all still works with 8.0 - don't expect any problems here btw.

Reply:
I had to bump it because it wasn't working before but it is now

Reply:
Yup, we standardised on 5.7 as newer versions weren't working at the time. But maybe that's fixed now :)